handler/handler.go

```diff
@@ -375,7 +375,15 @@ func (h *Handler) Run() error {
 				ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
 				if err = h.Execute(ctx, out, opt, h.lastPrefix, h.last, forceBatch); err != nil {
 					lastErr = WrapErr(h.last, err)
-					fmt.Fprintln(stderr, "error:", err)
+					ON_ERROR_STOP, parseErr := strconv.ParseBool(env.All()["ON_ERROR_STOP"])
+					if parseErr != nil {
```

Comment:
Can this be checked before executing the query? Failing fast usually gives better UX.

Reply:
There are probably a few places ON_ERROR_STOP can be checked/parsed though I'm not super familiar with the usql code. It needs to be checked both on startup or in the case someone calls `\set` in interactive. Do you know if there's a good place to move this to offhand? I'll look as well.

Reply:
Not from the top of my head. I'm not sure if other boolean vars are validated or not.

Reply:
Ok so I added a mechanism for optional param validation functions and added a case for ON_ERROR_STOP. One thing to note is that if the value is being set on the command line to an invalid value it does not get set but the error is silent. Here's what it looks like in practice:

```sh
[pbennes@localhost usql]$ ./usql -v ON_ERROR_STOP=asdf "snowflake://redacted" 
Connected with driver snowflake (<unknown>)
Type "help" for help.

sf:admin@127=> \set
ON_ERROR_STOP = 'false'

sf:admin@127=> \set ON_ERROR_STOP 1
sf:admin@127=> \set ON_ERROR_STOP 0
sf:admin@127=> \set ON_ERROR_STOP asdf
error: invalid value asdf for var ON_ERROR_STOP
```